### PR TITLE
Fix git_mirror env example defaults

### PR DESCRIPTION
**Pull request by greghaynes**
The git_mirror and git_update values are not valid in the openstack
block, they should be part of openstack_source.

### CODE REVIEW

**Comment by j2sol** [comment]
ok to test

**Comment by timrchavez** [comment]
retest

File: envs/example/defaults-2.0.yml [hunk 1]
```diff
@@ -95,17 +95,17 @@ monitoring:
...
   pip_trusted: pypi-mirror.openstack.blueboxgrid.com
   gem_sources:
     - https://gem-mirror.openstack.blueboxgrid.com
-  git_mirror:  https://github.com/openstack
-  git_update: yes
   ubuntu_mirror: https://apt-mirror.openstack.blueboxgrid.com/archive.ubuntu.com/ubuntu
 
+openstack_source:
+  git_mirror:  https://github.com/openstack
+  git_update: no
```
Comment (j2sol) [inline envs/example/defaults-2.0.yml]: Why did this change from yes to no?

**Comment by j2sol** [comment]
retest

**Comment by j2sol** [comment]
retest

**Comment by paulczar** [comment]
closing all PRs with no recent activity.

**Pull request closed without merging**